test/integration/narrow/app/logger.test.js

```diff
@@ -0,0 +1,10 @@
+describe('Logger test', () => {
```

Comment:
Not sure I see much value in this test.

Reply:
I do agree, but logger is mocked everywhere else and without this test the coverage dropped below 90%. I could write a test that uses the real logger if you want?